pria_lifechem/models/dnn_rf.py

```diff
@@ -1,455 +0,0 @@
-import argparse

```

Comment:
`dnn_rf.py` is mentioned in `pria_lifechem/models/README.md`.  Please delete that reference to say:
> Note that the tree_net.py model was not used in our manuscript.